### ISSUE TITLE
Physical impairment

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6729]
**Preferred term label:**
Diminished physical functioning

**Synonyms**
Diminished physical health
Decline in physical functional health

**Definition (free text, please give PubMed ID)**
A reduction in the ability to perform activities of daily living because of illness of functional deficits as compared to previous abilities. The 36-item Short Form (SF-36) health survey questionnaire is one of many methods used to measure patients' perceptions of diminished physical functioning.

**Parent term (use hpo.jax.org/app)**
Constitutional symptom HP:0025142

Or consider new term,
Diminished health-related quality of life 
A reduction in an individual's subjective assessment of his or her sense of well-being and ability to perform social roles.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
